packages/dart/sshnoports/bundles/universal.sh

```diff
@@ -659,7 +659,12 @@ client() {
       if [ -z "$device_name" ]; then
         done_input=true
       else
-        devices="$devices,$device_name"
+        if ! echo "$device_name" | grep -Eq '^[a-z][a-z0-9_]*$'; then
```

Comment:
instead of `*` can you change to `{1,15}`, we enforce a maximum of 15 chars

Reply:
fixed [18cdf89](https://github.com/atsign-foundation/noports/pull/1085/commits/18cdf8913f8d49e36e1ca41146b8af2e7ed84123)